src/webtransport.js

```diff
@@ -168,15 +229,21 @@ class Http3WTStream {
       const res = this.pendingres
       this.pendingoperation = null
       this.pendingres = null
-      res()
```

Comment:
Should error, because it indicates a bug in the implementation

Reply:
Or no `this.pendingoperation` guards this already, does `tsc` require an explicit type checking?